### ISSUE TITLE
Train the model for face parsing problem

### ISSUE THREAD

**Issue by myusernamesxy** [shekkizh/FCN.tensorflow#118]
I use this project to train my dataset, but the loss is always NAN. I labeled about 100 faces and produce the gray image, my class is 9, I tried to reduce learning rate and change the number of class, it still doesn't work.

**Comment by TauLau** [comment]
> I use this project to train my dataset, but the loss is always NAN. I labeled about 100 faces and produce the gray image, my class is 9, I tried to reduce learning rate and change the number of class, it still doesn't work.

Could you tell me how to use my own data set?My email address is 984471465@qq.com    Thanks!

**Comment by kunyu17** [comment]
could u tell me how to use my own dataset?  my email is zkyase@buaa.edu.cn. Thanks very much!!
